src/main/java/frc/robot/subsystems/elevator/P2025ElevatorIO.java

```diff
@@ -0,0 +1,106 @@
+            LinearSystemId.createDCMotorSystem(DCMotor.getKrakenX60(1),
+                    0.001, 1.0),
+            DCMotor.getKrakenX60(1).withReduction(1.0)
+    );
+
+    public P2025ElevatorIO(TalonFX motor, TalonFX motor2, int motor1Id) {
+        this.motor = motor;
+        this.motor2 = motor2;
+
+        TalonFXConfiguration config = new TalonFXConfiguration();
+
+        config.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+
+        config.Slot0.kV = 0.115;
+        config.Slot0.kA = 0.01;
+        config.Slot0.kP = 0.5;
+        config.Slot0.kI = 0.0;
+        config.Slot0.kD = 0.0;
+
+        config.MotionMagic.MotionMagicCruiseVelocity = 5500.0 / 60.0;
+        config.MotionMagic.MotionMagicAcceleration = 2500.0 / 60.0;
+
+        motor.getConfigurator().apply(config);
+        motor2.getConfigurator().apply(config);
+
+        motor2.setControl(new Follower(motor1Id, true));
+    }
+
+    @Override
+    public void updateInputs(ElevatorInputs inputs) {
+        // Sims for Motor 1
+        var simStateMotor1 = motor.getSimState();
+        simMotor1.setInputVoltage(simStateMotor1.getMotorVoltage());
+        //Updates the sim information every 20 ms
+        simMotor1.update(Robot.kDefaultPeriod);
+        simStateMotor1.setRotorAcceleration(simMotor1.getAngularAcceleration());
+        simStateMotor1.setRotorVelocity(simMotor1.getAngularVelocity());
+        simStateMotor1.setRawRotorPosition(simMotor1.getAngularPosition());
+
+        // Sims for Motor 2
+        var simStateMotor2 = motor2.getSimState();
+        simMotor2.setInputVoltage(simStateMotor2.getMotorVoltage());
+        //Updates the sim information every 20 ms
+        simMotor2.update(Robot.kDefaultPeriod);
+        simStateMotor2.setRotorAcceleration(simMotor2.getAngularAcceleration());
+        simStateMotor2.setRotorVelocity(simMotor2.getAngularVelocity());
+        simStateMotor2.setRawRotorPosition(simMotor2.getAngularPosition());
+
+        // Inputs for Motor 1
+        inputs.currentVelocityMotor1 = Units.rotationsToRadians(motor.getVelocity().getValueAsDouble());
+        inputs.currentAppliedVoltageMotor1 = motor.getMotorVoltage().getValueAsDouble();
+        inputs.currentPositionMotor1 = Units.rotationsToRadians(motor.getPosition().getValueAsDouble());
+        inputs.motorTemperatureMotor1 = motor.getDeviceTemp().getValueAsDouble();
+        inputs.currentDrawMotor1 = motor.getSupplyCurrent().getValueAsDouble();
+
+        // Inputs for Motor 1
+        inputs.currentVelocityMotor2 = Units.rotationsToRadians(motor2.getVelocity().getValueAsDouble());
+        inputs.currentAppliedVoltageMotor2 = motor2.getMotorVoltage().getValueAsDouble();
+        inputs.currentPositionMotor2 = Units.rotationsToRadians(motor2.getPosition().getValueAsDouble());
+        inputs.motorTemperatureMotor2 = motor2.getDeviceTemp().getValueAsDouble();
+        inputs.currentDrawMotor2 = motor2.getSupplyCurrent().getValueAsDouble();
```

Comment:
Your conversion from motor rotations to mechanism position isn't correct here. Instead of converting to radians, we actually want to convert to a distance in meters.

Elevators have a pulley which converts the angular movement of the gearbox into linear movement of the elevator. Once you get to this part in the review we'll review the math for the conversions together.